torchx/lib/torchx/nif.ex

```diff
@@ -3,7 +3,7 @@ defmodule Torchx.NIF do
   @on_load :__on_load__
 
   def __on_load__ do
-    path = :filename.join(:code.priv_dir(:torchx), 'torchx')
+    path = Path.join([:code.priv_dir(:torchx), 'torchx'])
```

Comment:
load_nif prefers a char list, I think we should keep this code as is rather. :)

Reply:
Forgot to rollback this part :sweat_smile: